spring-web-modules/spring-thymeleaf/src/main/java/com/baeldung/thymeleaf/config/WebApp.java

```diff
@@ -1,9 +1,11 @@
 package com.baeldung.thymeleaf.config;
 
-import javax.servlet.ServletRegistration.Dynamic;
+
```

Comment:
Can you remove the empty lines here please.

Reply:
Done